libopenage/renderer/demo/demo_0.cpp

```diff
@@ -9,6 +9,7 @@
 #include "renderer/resources/mesh_data.h"
 #include "renderer/resources/shader_source.h"
 #include "renderer/shader_program.h"
+#include "renderer/demo/util.h"
```

Comment:
Have you added autoformatting with `clang-format` to your editor? The imports should be sorted alphabetically if you use `clang-format`.